src/init/features/dataconnect/index.ts

```diff
@@ -55,7 +56,19 @@ const defaultConnector = {
   ],
 };
 
+// doSetup is split into 2 phases - ask questions and then actuate files and API calls based on those answers.
 export async function doSetup(setup: Setup, config: Config): Promise<void> {
+  const info = await askQuestions(setup, config);
```

Comment:
In this setup, vscode will need both a doSetup function and a askQuestions function. Is this the expectation? I was thinking we could limit VSCode to need an askQuestions function, which has a set output type.

Reply:
actuate is also exported - I'd suggest the VSCode implements it's own askQuestions() that returns RequiredInfo, and then directly call actuate